package.json

```diff
@@ -4,11 +4,15 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "format": "npx prettier --write ."
```

Comment:
I like that you've added .prettierrc rules, and added this command to your package.json to format your code. If you ran this command, `npm run format`, before submitting this code review it would have fixed most of the problems you have with formatting